plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/MetricsWrapper.java

```diff
@@ -40,6 +41,7 @@ public MetricsWrapper(
                 columnSizes,
                 valueCounts,
                 nullValueCounts,
+                ImmutableMap.of(), // Trino CBO doesn't utilise nanValueCounts at all
```

Comment:
These are Iceberg metrics that are part of the Iceberg specification, and thus we should be writing them. The purpose of `MetricsWrapper` is to allow JSON serialization of Iceberg `Metrics`, so we should be pass the value through here. The Parquet writer already collects NaN counts, and the ORC writer should be updated to do so.

Saying "Trino doesn't use them so we don't care" is hostile.